platform-emf/src/main/java/com/softicar/platform/emf/management/importing/variable/find/EmfImportVariablesFinder.java

```diff
@@ -18,11 +21,13 @@ public EmfImportVariablesFinder(List<List<String>> rows) {
 
 	public SetMap<String, EmfImportVariableCoordinates> find() {
```

Comment:
I think this method (and the class itself) deserves some Javadoc.
E.g., personally, I was surprised that we silently ignore malformed variable names.